packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2474,6 +2485,62 @@ describe('plugin-meetings', () => {
             reason: MEETING_REMOVED_REASON.CLIENT_LEAVE_REQUEST,
           });
         });
+
+        it('should send client.call.leave after meetingRequest.leaveMeeting', async () => {
+          const leave = meeting.leave();
+
```

Comment:
i think it would be better to check it's not called before awaiting, and check it's called after